client/jetpack-cloud/components/guided-tour/index.tsx

```diff
@@ -141,7 +151,14 @@ const GuidedTour = ( { className, tours, preferenceName }: Props ) => {
 			position={ popoverPosition }
 		>
 			<h2 className="guided-tour__popover-heading">{ title }</h2>
-			<p className="guided-tour__popover-description">{ description }</p>
+			<p className="guided-tour__popover-description">
+				{ description.split( '\n' ).map( ( line, index ) => (
+					<React.Fragment key={ index }>
+						{ line }
+						{ index < description.split( '\n' ).length - 1 && <br /> }
+					</React.Fragment>
```

Comment:
Instead of `<React.Fragment></React.Fragment>`, just use `<></>`.

Reply:
When using `<></>` we were getting a React warning about missing unique key prop. I fixed this by adding the key prop, which needs the `React.Fragment` component declared fully. This the commit that fixed the warning: https://github.com/Automattic/wp-calypso/pull/85488/commits/817d7edb28ae9bced83654c08e9194b81d4792f2

Reply:
Also, shouldn't we be passing the <br /> directly to the description rather than passing newlines?

Reply:
This change was added: 4979eeac816e26b4b3989a8a9798c73ca2f3dda3